src/commands/show.ts

```diff
@@ -34,10 +36,10 @@ export async function searchProblem(): Promise<void> {
     if (!choice) {
         return;
     }
-    await showProblemInternal(choice.value);
+    await showProblemInternal(problems.find((problem: IProblem) => problem.id === choice.value) as IProblem);
```

Comment:
Have you tried to change the line 118 from 
`value: problem.id,` to `value: problem`?

Reply:
emm...Because of type error, so I use IProblem on both sides.

Reply:
I mean, you are using `problems.find((problem: IProblem) => problem.id === choice.value)` to get the problem, that is because the `choice.value` only stores the `id`. You can make it to store the problem itself.